src/sage/databases/findstat.py

```diff
@@ -1499,7 +1499,8 @@ def __call__(self, elt):
             from sage.repl.preparse import preparse
             try:
                 l = {}
-                code = "from sage.all import *\n" + preparse(self.sage_code())
+                environment = 'sage.all'
+                code = f"from {environment} import *\n" + preparse(self.sage_code())
```

Comment:
Same comment as in the other PR...

Reply:
Likewise.

Reply:
this reads cryptic to me - what's the issue here?

Reply:
Sorry for the missing details. See https://github.com/sagemath/sage/pull/35105. My issue is that this is a (almost trivial, but well) runtime change to cover up a bug in the regex that is used to detect this all imports.